src/_component/open-source/open-source.pug

```diff
@@ -0,0 +1,50 @@
+- var sectionTitle = data['open-source'].sectionTitle
```

Comment:
instead of using in the view all the time ```data[`open-source`].stats.title``` or others, you can extract the variable here, and use them later

Reply:
```
- 
  var sectionTitle = data['open-source'].sectionTitle
  var title = data[`open-source`].stats.title
  var projectName = data[`open-source`].projects.projectName
  var project = data[`open-source`].projects.project  
```